### ISSUE TITLE
[steamshovel] script execution broken in python3 (Trac #2370)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2362]
Apparently python3 does not support `execfile()`, breaking automatic execution of scripts using the `-s` command line argument.

Migrated from https://code.icecube.wisc.edu/ticket/2370
```json
{
    "status": "closed", 
    "changetime": "2020-07-15T23:13:13", 
    "description": "Apparently python3 does not support `execfile()`, breaking automatic execution of scripts using the `-s` command line argument.", 
    "reporter": "karg", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1594854793115010", 
    "component": "combo core", 
    "summary": "[steamshovel] script execution broken in python3", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2019-11-03T14:20:56", 
    "milestone": "Autumnal Equinox 2020", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20191211T15:34:25: 
* olivas changed component from "other" to "icecube offline"
* olivas changed status from "new" to "accepted"
* olivas changed owner from "" to "olivas"
* olivas changed priority from "normal" to "critical"
* olivas changed milestone from "" to "Winter Solstice 2019"

**Comment by icecube-trac** [comment]
Trac update at 20191218T20:26:01: 
* olivas commented:

> We're likely not going to be able to fully support Python3 this release.


* olivas changed milestone from "Winter Solstice 2019" to "Vernal Equinox 2020"

**Comment by icecube-trac** [comment]
Trac update at 20200217T19:53:21: olivas changed summary from "steamshovel script execution broken in python3" to "[steamshovel] script execution broken in python3"

**Comment by icecube-trac** [comment]
Trac update at 20200526T18:37:00: jvansanten commented:

> [https://stackoverflow.com/a/437857 The internets say] this is a fairly trivial fix. Anyone want to try it out?

**Comment by icecube-trac** [comment]
Trac update at 20200624T12:31:42: 
* olivas changed milestone from "Vernal Equinox 2020" to "Autumnal Equinox 2020"
* olivas commented:

> Milestone Vernal Equinox 2020 deleted

**Comment by icecube-trac** [comment]
Trac update at 20200715T23:13:13: 
* olivas commented:

> In changeset 181104:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="181104"
> scripting with python3 and w/o ipython should work again. closes #2362.
> ```


* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"
